build/jenkins/jobs/test/unit_test_runner.dsl

```diff
@@ -0,0 +1,38 @@
+job('unit-test-runner') {
+
+    configure { project ->
+        project / 'properties' / 'hudson.plugins.copyartifact.CopyArtifactPermissionProperty' / 'projectNameList' {
+            'string' "*rook*"
+        }
+    }
+
+    parameters {
+            stringParam('WORKSPACE', '')
+
+    }
+
+    customWorkspace('${WORKSPACE}')
+
+    throttleConcurrentBuilds {
+        maxPerNode(1)
+        maxTotal(1)
+    }
+
+    //// not yet handling test artifacts
+    // publishers {
+    //     archiveArtifacts {
+    //         pattern('results/*')
+    //         onlyIfSuccessful()
+    //     } 
+    // }
+
+    steps {
+
+        // execute test 
+        shell('(build/run make check -j${parallel})')
+
+    }
+
+    concurrentBuild false
```

Comment:
concurrent CI and PR builds are disabled at this time because of [issue 202](https://github.com/rook/rook/issues/202) where we are using a single shared build slave

we would also need update the DSL to create unique workspaces for each CI/PR run so they don't clobber each other; currently each CI and PR run reuses the same workspace location on each invocation